pages/patterns/delete.js

```diff
@@ -1,108 +1,126 @@
-      ),
-    },
-  ]);
+// To change what appears on the Delete Pattern page, edit the data below.
+
+const severity = 'medium';
+const pageHeading = 'Services';
+const tableDescription =
+  'Services enable you to organize and identify your conversations by use case, or manage them in multiple environments (e.g. dev, stage, prod). This information held within a service is siloed, protecting both your recipients&apos; data.';
+const emptyState = 'Nothing to delete here!';
+const columns = ['friendlyName', 'sid', 'greetings'];
+const rows = [
+  [
+    <Text as="span" fontFamily="fontFamilyText">
+      Fiiiiiiiiirst thing
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyCode">
+      4392908903
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyText">
+      hi
+    </Text>,
+  ],
+  [
+    <Text as="span" fontFamily="fontFamilyText">
+      Second Thing
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyCode">
+      8439025738
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyText">
+      hello
+    </Text>,
+  ],
+  [
+    <Text as="span" fontFamily="fontFamilyText">
+      Third thing
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyCode">
+      8439025738
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyText">
+      Welcome!
+    </Text>,
+  ],
+  [
+    <Text as="span" fontFamily="fontFamilyText">
+      Fourth this
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyCode">
+      6239485739
+    </Text>,
+  ],
+];
+
+export default function Delete() {
+  const [tableRows, setTableRows] = useState(rows);
 
   const toaster = useToaster();
   const [isOpen, setIsOpen] = React.useState(false);
-  const [serviceToDelete, setServiceToDelete] = React.useState({});
+  const [serviceToDelete, setServiceToDelete] = React.useState([]);
 
-  const deleteService = (serviceObj) => {
+  const deleteService = (serviceRow) => {
     toaster.push({
-      message: `${serviceObj.friendlyName.props.children} was successfully deleted.`,
+      message: `${serviceRow[0].props.children} was successfully deleted.`,
       variant: 'success',
       dismissAfter: 3000,
     });
-    const updatedServices = data.filter((service) => service.id !== serviceObj.id);
-    setData(updatedServices);
-    setServiceToDelete({});
+    const updatedServices = tableRows.filter((row) => tableRows.indexOf(serviceRow) !== tableRows.indexOf(row));
+    setTableRows(updatedServices);
+    setServiceToDelete([]);
   };
 
-  const handleDelete = (serviceObj) => {
+  const handleDelete = (serviceRow) => {
     if (severity === 'low') {
-      deleteService(serviceObj);
+      deleteService(serviceRow);
     } else if (severity === 'medium') {
       setIsOpen(true);
-      setServiceToDelete(serviceObj);
+      setServiceToDelete(serviceRow);
+    } else {
+      throw new Error(
+        'No severity level was detected. Go back to delete.js and make sure to change severity to "low", "medium", or "high"!',
+      );
```

Comment:
Another great error message! Maybe we should take out 'high' as an option in this message though, so that they don't type that in while it's not working?

Reply:
good idea. just removed it, and we can add it later when the high severity pattern is available. thanks!!